apps/web/components/getting-started/steps-views/UserSettings.tsx

```diff
@@ -112,10 +112,16 @@ const UserSettings = (props: IUserSettingsProps) => {
       </div>
       <Button
         type="submit"
-        className="mt-8 flex w-full flex-row justify-center"
+        className="mt-8 flex w-full  flex-row justify-center "
         disabled={mutation.isPending}>
-        {t("next_step_text")}
-        <ArrowRight className="ml-2 h-4 w-4 self-center" aria-hidden="true" />
+        {mutation.isPending ? (
+          <div className="cursor-wait px-[220px] py-2  "> Processing...</div>
+        ) : (
+          <>
+            {t("next_step_text")}
+            <ArrowRight className="ml-2 h-4 w-4 self-center" aria-hidden="true" />
+          </>
+        )}
```

Comment:
you just have to use loading property in Button as mentioned by peer here https://github.com/calcom/cal.com/pull/14028#issuecomment-1988039296

Reply:
yes sir "use loading={mutaiton.isPending}" i am only using this property not use "usestate"

Reply:
![image](https://github.com/calcom/cal.com/assets/50249571/eec26ef3-4f8c-4aca-a3ed-08eb7ef9d726)
i am working on these things

Reply:
@murtaza030 you have to pass loading property to Button and don't use this

 {mutation.isPending ? (
          <div className="cursor-wait px-[220px] py-2  "> Processing...</div>
        ) : (
          <>
            {t("next_step_text")}
            <ArrowRight className="ml-2 h-4 w-4 self-center" aria-hidden="true" />
          </>
        )}

Reply:
never mind i have fixed it now